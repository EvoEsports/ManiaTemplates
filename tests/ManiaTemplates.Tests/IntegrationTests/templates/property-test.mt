<component>
    <import component="TestComponent" as="TestComponent" />
    <import component="Wrapper" as="Wrapper" />
    
    <property type="string" name="testVariable" />
    
    <template>
        <TestComponent text="{{ testVariable }}test" />
        <TestComponent text="{{ testVariable }}{{ testVariable }}" />
        <TestComponent text="test" />
        <label text="{{ testVariable }}" />
        <Wrapper width="{{ 123.0 }}">
            <Wrapper width="{{ 77.0 }}">
                <label text="{{ testVariable }}" />
            </Wrapper>
        </Wrapper>
    </template>
</component>