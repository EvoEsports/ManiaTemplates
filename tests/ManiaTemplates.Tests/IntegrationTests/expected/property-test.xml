<manialink version="3" id="MtPropertyTest" name="EvoSC#-MtPropertyTest">
<frame size="15 11">
<label size="15 3" text="integrationtest" />
</frame>
<frame size="15 11">
<label size="15 3" text="integrationintegration" />
</frame>
<frame size="15 11">
<label size="15 3" text="test" />
</frame>
<label text="integration" />
<frame size="123 11">
<frame size="77 11">
<label text="integration" />
</frame>
</frame>
</manialink>
